nixos/modules/security/pam.nix

```diff
@@ -861,6 +861,7 @@ in
       unix_chkpwd = {
         source = "${pkgs.pam}/sbin/unix_chkpwd.orig";
         owner = "root";
+        group = "nogroup";
```

Comment:
Similarly here with `nogroup`.

Reply:
this one is setuid root so we can make it owned by root:root I think.

Reply:
Done